crates/uv/tests/run.rs

```diff
@@ -2136,37 +2136,21 @@ fn run_remote_pep723_script() {
         r"(?m)^Reading inline script metadata from:.*\.py$",
         "Reading inline script metadata from: [TEMP_PATH].py",
     ));
-    uv_snapshot!(filters, context.run().arg("https://gist.githubusercontent.com/manzt/cb24f3066c32983672025b04b9f98d1f/raw/d675b26fe8b3a8e301705af80a53fe5de15a0273/pep723.py"), @r###"
+    uv_snapshot!(filters, context.run().arg("https://raw.githubusercontent.com/astral-sh/uv/2b7997/scripts/uv-run-remote-script-test.py").arg("CI"), @r###"
```

Comment:
should we point to main here?

Reply:
No I think the commit revision like you have it here is the right way to go. But I would use the full revision hash.

Reply:
Done.